### PR TITLE
Has Skill return value

### PR DESCRIPTION
**Pull request by Sicprofundus**
- corrected inline bool return value from "0" to false

### CODE REVIEW

**Comment by Sicprofundus** [comment]
@brainiac sorry for deleting and recommitting, something happened with my git this morning i had to fix.

no clue where the asd came from, but i appreciate your eyeballs

File: src/main/MQ2Inlines.h [hunk 1]
```diff
@@ -249,7 +249,7 @@ inline int GetFocusRangeModifier(const EQ_Spell* pSpell, ItemPtr& pItemOut)
 
 inline bool HasSkill(int nSkill)
 {
-	return pLocalPC ? pLocalPC->HasSkill(nSkill) : 0;
+	return pLocalPC ? pLocalPC->HasSkill(nSkill) : false;
```
Comment (brainiac) [inline src/main/MQ2Inlines.h:252]: this can be written without the ternary `return pLocalPC && pLocalPC->HasSkill(nSkill)`

Comment (Sicprofundus) [reply src/main/MQ2Inlines.h]: I appreciate you pointing that out, sometimes i have a bad habit of not stepping back for the bigger "what are we trying to do" and instead see a small part of it

Comment (dannuic) [reply src/main/MQ2Inlines.h]: what _are_ you trying to do? This isn't a functional change

Comment (Sicprofundus) [reply src/main/MQ2Inlines.h]: it was returning a  "0" for a bool function, so i was trying to clean it up to return false instead of 0

**Pull request merged**